### ISSUE TITLE
changeAnnotations event doesn't emit

### ISSUE THREAD

**Issue by TimChinye** [mkslanc/ace-linters#133]
I have a function that is called when a change is made and if there are no errors after the change.

The recent update makes it such that `changeAnnotations` events is never emitted when the code is changed.

The event is triggered when you use `.setAnnotations()`, but that's seemingly it.

There's not much code I can show here, except:
```js
    leftEditor.getSession().on('change', () => {
      console.log('change');
    });
    leftEditor.getSession().on('changeAnnotation', () => {
      console.log('changeAnnotation');
});
```
That changeAnnotation is just not logged.

When I went back from @latest to @1.2.3, the issue was immediately fixed.

I'm using the CSS mode.

**Comment by mkslanc** [comment]
@TimChinye I can't reproduce this issue, do you have more context to share?
```javascript
    editor.getSession().on('changeAnnotation', () => {
        console.log('changeAnnotation');
    });
```
 
[screen-capture (66).webm](https://github.com/user-attachments/assets/48e75b66-345f-44b1-8914-bceabb48fb62)
